app/scripts/controllers/metametrics.test.js

```diff
@@ -64,23 +64,24 @@ const DEFAULT_PAGE_PROPERTIES = {
   ...DEFAULT_SHARED_PROPERTIES,
 };
 
-function getMockNetworkController(
-  chainId = FAKE_CHAIN_ID,
-  provider = { type: NETWORK_TYPES.MAINNET },
-) {
-  let networkStore = { chainId, provider };
+function getMockNetworkController() {
+  let state = {
+    provider: {
+      type: NETWORK_TYPES.GOERLI,
+      chainId: FAKE_CHAIN_ID,
+    },
+    network: 'loading',
+  };
   const on = sinon.stub().withArgs(NETWORK_EVENTS.NETWORK_DID_CHANGE);
   const updateState = (newState) => {
-    networkStore = { ...networkStore, ...newState };
```

Comment:
This test helper was setting up state for this fake network controller in a way that didn't match how the real network controller sets up state. These changes should bring them closer in alignment.